ezResx/Errors/DataLossException.cs

```diff
@@ -1,20 +1,39 @@
-﻿using System;
+﻿using ezResx.Data;
+using System;
+using System.Collections.Generic;
 
 namespace ezResx.Errors
 {
     internal class DataLossException : Exception
     {
-        public DataLossException()
+        public List<ResourceItem> MissingData { get; set; }
```

Comment:
IEnumerable could be used? MissingData should have a public getter and private setter.